hack/verify-gherkin.sh

```diff
@@ -22,8 +22,10 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
+pip3 install reformat-gherkin==3.0.1 --ignore-installed PyYAML
```

Comment:
TODO: consider using a virtual env for the repo so we don't need to depend on the user's system packages.

https://docs.python.org/3/library/venv.html

Reply:
Thanks @BenTheElder, I opened a separate issue for this and will do it after this one

Reply:
https://github.com/kubernetes-sigs/ingress-controller-conformance/issues/110